File: app/src/main/java/com/artmurka/artmurkaapp/Model/InterfacesModel/IAllRequestObservers.java
package com.artmurka.artmurkaapp.Model.InterfacesModel;

import com.artmurka.artmurkaapp.Model.Pojo.ItemList.Categories.Example;

import io.reactivex.Observable;


public interface IAllRequestObservers {
    Observable<Example> getCategories();
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/interfacesmodel/IAllRequestObservers.kt
package com.artmurka.artmurkaapp.data.model.interfacesmodel

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.categories.Example

import io.reactivex.Observable


interface IAllRequestObservers {
    val categories: Observable<Example>
}
